src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/.template.config/localize/templatestrings.cs.json

```diff
@@ -4,6 +4,9 @@
   "description": "Šablona projektu pro vytvoření aplikace Blazor, která běží ve WebAssembly. Tato šablona se dá využít pro webové aplikace s propracovanými dynamickými uživatelskými rozhraními (UI).",
   "symbols/Framework/description": "Cílová architektura pro projekt",
   "symbols/Framework/choices/net8.0/description": "Cílový net8.0",
+  "symbols/IncludeSampleContent/displayName": "_Include sample pages",
```

Comment:
How do these eventually get translated?

Reply:
There's a system. Eventually the system will make a PR that updates the newly-added strings to translated values. Nothing else has to be done for it in this PR.